tests/acceptance/FormEditorAddDividerCest.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+namespace MailPoet\Test\Acceptance;
+
+use MailPoet\Test\DataFactories\Form;
+use MailPoet\Test\DataFactories\Segment;
+
+class FormEditorAddDividerCest {
+  public function addDividerBlock(\AcceptanceTester $i) {
+    $i->wantTo('Add divider block to the editor');
+    $segmentFactory = new Segment();
+    $segmentName = 'Fancy List';
+    $segment = $segmentFactory->withName($segmentName)->create();
+    $formName = 'My fancy form';
+    $form = new Form();
+    $form->withName($formName)->withSegments([$segment])->create();
+
+    $i->login();
+    $i->amOnMailPoetPage('Forms');
+    $i->waitForText($formName);
+    $i->clickItemRowActionByItemName($formName, 'Edit');
+    $i->waitForElement('[data-automation-id="form_title_input"]');
+
+    // Add divider & spacer block
+    $i->addFromBlockInEditor('Divider / Spacer');
+
+    // Check and modify divider & spacer block
```

Comment:
I still think that this is a perfect example of comment that could be place into $i->wantTo function.

Reply:
Done.